### DISCUSSION TITLE
🎉🎉🎉 v.3.1.0 - short update what we are doing

### DISCUSSION THREAD

**Discussion by Blackymas** [Blackymas/NSPanel_HA_Blueprint#252] [Announcements]
💥💥💥 hi @ all 💥💥💥

#### We have added a few cool things like

- Qr-Code 
- Fallback for Relay
- display current versions of Esphome, TFT, Blueprint and IP in the boot menu
- Entity Cards - we are working on it right now
- we have also made some improvements in the code.

#### Additionally the Wiki was updated - like the first setup. PLEASE HAVE A LOOK! We change the TFT Upload from Github to local. But don't worry via Github will still work. But the local TFT upload will fix the problem with the white screens etc.

Because the Entity Cards **_( 4 pages with 8 entries so altogether place for 32 sensors etc)_** costs us much time - therefore the next update can last still 2-3 weeks.

But the wait is worth it - promised ;)

#### In the meantime you would have enough time to leave a donation, we would be very happy about any donation :)

[![Paypal](https://user-images.githubusercontent.com/41958506/212499642-b2fd097a-0938-4bfc-b37b-74df64592c58.png)](https://www.paypal.com/donate/?hosted_button_id=S974SWQMB8PB2)

**Comment by Blackymas** [comment]
am so happy about the project and the community ;)

**Reply by pwirkus** [reply]
I donated for the great work. Thank you, I'm waiting for the latest version :)

**Comment by pwirkus** [comment]
good morning, would a friend tell me how to upload this version? v3.1.0? greetings and thank you

**Reply by danw87** [reply]
> good morning, would a friend tell me how to upload this version? v3.1.0? greetings and thank you

Hi, this version is still in development, it will be available soon.

Thanks

**Comment by pwirkus** [comment]
danw87, Thank you :)

**Comment by stevoh6** [comment]
im already testing it, its awesome ;)

**Reply by Blackymas** [reply]
dev Version 3.1.0?

**Reply by stevoh6** [reply]
yep ;) looking good already, also I open discussion about layout,  because I was thinking about other and also mine ise cases.

**Reply by pwirkus** [reply]
where is ? or when will it be? version 3.1 available?
